.github/BUILD.bazel.tmp

```diff
@@ -0,0 +1,54 @@
+# Copyright © 2025 Apple Inc. and the Pkl project authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@aspect_bazel_lib//lib:copy_to_directory.bzl", "copy_to_directory")
+load("@aspect_bazel_lib//lib:write_source_files.bzl", "write_source_files")
+load("//pkl:defs.bzl", "pkl_eval")
+
+pkl_eval(
+    name = "index",
+    srcs = ["index.pkl"],
+    multiple_outputs = True,
+    deps = ["@pklproject_github_actions//:packages"],
+    # TODO need to provide "PWD" as an env var
+    # env = {
+    #     "PWD": ".",
+    # },
```

Comment:
@KushalP FYI: I've disabled the bazel side of things here.

With the new version of pkl.impl.ghactions, we need to pass the PWD to Pkl so that it can read the existing lockfile (see https://github.com/apple/rules_pkl/pull/142/files#diff-2d8c57efa466feee81dd1a2e97d11a3593578baf10328e6a2616227fe068671d).

However, it looks like we're missing the ability to set env vars in `pkl_eval` right now.